comparisontool/static/comparisontool/js/comparison-tool.js

```diff
@@ -1694,32 +1534,30 @@ var CFPBComparisonTool = (function() {
             // Pie Charts
             var x;
             for (x = 1; x <= 3; x++ ) {
-                pies[x] = Raphael($("[data-column='" + x + "'] .debt-pie")[0], 125, 125);
-                pies[x].circle(62, 62, 50);
-                circles[x] = pies[x].circle(62, 62, 50);
-                circles[x].attr({fill: "Gray", stroke: "White", "stroke-width": 2});
-                loans[x] = pies[x].path("M 62 62");
-                loans[x].attr({fill: "Red", stroke: "White", "stroke-width": 2});   
-            }
-
-            // Default Rate Bars
-            for (x = 1; x <= 3; x++ ) {
-                bars[x] = Raphael($("[data-column='" + x + "'] .default-rate-chart")[0], 200, 100);
-                var bottomline = bars[x].path("M 0 100 L 200 100");
-                bottomline.attr({"stroke": "#585858", "stroke-width": 3})
-                averagebars[x] = bars[x].rect(120, 50, 60, 50);
-                averagebars[x].attr({"fill":"#585858", "stroke": "#585858"});
-                defaultbars[x] = bars[x].rect(20, 100, 60, 0);
-                defaultbars[x].attr({"fill":"#585858", "stroke": "#585858"});
-            }
-
-            // Borrowing meter
-            for (x = 1; x <= 3; x++ ) {
-                meters[x] = Raphael($("[data-column='" + x + "'] .median-borrowing-chart")[0], 200, 100);
-                var circle = meters[x].circle(101, 100, 8);
-                circle.attr({"stroke": "#585858", "stroke-width": 1, "fill": "#585858"});
-                meterarrows[x] = meters[x].path("M 100 100 L 50 100");
-                meterarrows[x].attr({"stroke": "#f5f5f5", "stroke-width": 2});
+                pies[x] = document.createElementNS( 'http://www.w3.org/2000/svg', 'svg' );
```

Comment:
I'm not sure there is any reason to populate `pies` here – what is it getting us?

Reply:
That's a good call - this is legacy code that could be removed, but due to the amount of JS reorganization this repo needs, I'm just going to call this out-of-scope and pick it up when I modernize this repo.